src/lib.rs

```diff
@@ -55,7 +55,7 @@
 //! to the network through any node, and exchange public keys with it. That node becomes a
 //! bootstrap node for the client, and messages to and from the client will be routed over it.
 //!
-//! ```
+//! ```no_run
```

Comment:
Yes, it breaks, because mock crust service needs to be bound to the mock `Device` object, which is done using the `make_current(device, || { /* create Core here */ })` hack. Without it, the mock Service asserts. Not sure it's worth doing anything about it. The examples still need to compile anyway, even with the `no_run` annotation.